pkg/cmd/save.go

```diff
@@ -101,15 +110,86 @@ func save(c *cobra.Command) error {
 		}
 	}
 
-	// lastly save this binary...
-	me, _ := os.Executable()
-	if err := copyBin(me, saveDir); err != nil {
-		return fmt.Errorf(
-			"problem trying to save %s as %s/%s:%s",
-			me,
-			saveDir,
-			me,
-			err)
+	// Save the binary for the target platform
+	platform := c.Flag(FlagTargetPlatform).Value.String()
+	return saveMe(saveDir, platform)
+}
+
+// saveMe saves a copy of the target binary in the save dir
+func saveMe(saveDir string, platform string) error {
```

Comment:
`func saveMe(saveDir, platform string) error`

Reply:
Fixed this. (and added linters)